src/main/java/com/halotroop/arrchecker/ARRChecker.java

```diff
@@ -16,15 +17,23 @@
 
     @Override
     public void onInitialize() {
+        final List<String> ignore = ImmutableList.of(
+                "minecraft",
+                "fabric-language-scala",
+                "com_moandjiezana_toml_toml4j",
+                "java");
```

Comment:
Seems like a pretty arbitrary list.

Minecraft should certianly not be ignored. It should trigger the Minecraft is ARR warning.

Reply:
Minecraft isn't completely ignored anyways but I've for now removed all ignore lists and instead use the fabric provided "builtin" type to filter at least minecraft and java.